layers/+tools/file-manager/packages.vim

```diff
@@ -2,7 +2,7 @@ MP 'danro/rename.vim',               { 'on' : 'Rename' }
 
 " Refer to https://github.com/junegunn/dotfiles  vimrc
 MP 'scrooloose/nerdtree', { 'on': ['NERDTreeToggle', 'NERDTreeFind'] }
-autocmd! User nerdtree call spacevim#autocmd#nerdtree#Init()
+autocmd! User nerdtree call spacevim#autocmd#nerdtree#Init() | if exists('*UserConfig') | call UserConfig() | endif
```

Comment:
This is not neccessary as you can use `autocmd FileType nerdtree call YourNerdtreeConfig()` in `UserConfig()`.